src/manage_sql_report_formats.c

```diff
@@ -3551,9 +3550,8 @@ run_report_format_script (gchar *report_format_id,
       /* Just run the command as the current user. */
 
       ret = system (command);
-      /* Ignore the shell command exit status, because we've not
-        * specified what it must be in the past. */
-      if (ret == -1)
+      /* Report scripts should return 0 since version 21.04 */
+      if (ret)
```

Comment:
The return from system is either -1 or a "wait status".  Should you be checking for -1 and using WEXITSTATUS?

Reply:
I've changed this in e96859a84babecf7861ec7e1ef693b60333a0357